### ISSUE TITLE
Improved manual

### ISSUE THREAD

**Issue by gforge** [AlexMili/torch-dataframe#15]
The README is currently lagging. We need to cover:
- batch functionality
- exemplify how to set up a network using the dataframe
- the basic statistics available

... and more. I'm not sure if it's best to build a larger README or if we should start a doc-folder. Anyway the README requires a TOC to start with.

**Comment by AlexMili** [comment]
You're right, we can't let the README get bigger and bigger. I think we should keep the "doc" term (and directory) for the API documentation and use the WIKI for tutorials, examples, etc.

**Comment by gforge** [comment]
Sounds good. Do we have the WIKI outside the project or is the WIKI handled using regular PRs?

**Comment by AlexMili** [comment]
For now the regular Github WIKI will be fine.
